server/reflector/processors/types.py

```diff
@@ -16,6 +17,16 @@ class DiarizationSegment(TypedDict):
 
 
 PUNC_RE = re.compile(r"[.;:?!…]")
+SENTENCE_END_RE = re.compile(r"[.?!…]$")
+
+# Max segment length for words_to_segments() - breaks on any punctuation (. ; : ? ! …)
+# when segment exceeds this limit. Used for non-multitrack recordings.
+MAX_SEGMENT_CHARS = 120
+
+# Max segment length for words_to_segments_by_sentence() - only breaks on sentence-ending
+# punctuation (. ? ! …) when segment exceeds this limit. Higher threshold allows complete
+# sentences in multitrack recordings where speakers overlap.
+MAX_SENTENCE_SEGMENT_CHARS = 300
```

Comment:
Current implementation in the processor have 1000, would be nice to keep it here too. https://github.com/Monadical-SAS/reflector/blob/main/server/reflector/processors/transcript_liner.py